docs/guide.rst

```diff
@@ -92,6 +92,22 @@ Here's how you can deploy new contracts:
     :dedent: 4
 
 
+Deploying new contracts with Universal Deployer Contract (UDC)
```

Comment:
I think we should create a section about deploying and show this and deploy_account as 2 possible paths.

Reply:
Can be done as a separate PR.